docs/3.0.0-beta.x/plugins/documentation.md

```diff
@@ -171,6 +178,46 @@ To access your documentation on a custom path, you will have to update the `path
 }
 ```
 
+### Specify the plugin name seperator for the OPEN API components schema name and tags name
+
+To change the default plugin name seperator `-` that is used to generate your components schema name and tags name (e.g. plugin users-permissions [component schema name => `Users-PermissionsUser`, tag name => `UsersPermissions - User`]), you will have to update the `pluginNameSeperator` key. This parameter could be set to an empty character so it would generate a component schema name that is without any separator which is then looking like `UsersPermissionsUser` and becomes compatible with AWS API Gateway import API capability.
```

Comment:
I think we should just remove the `-` if it isn't supported instead of adding an option for it.

Reply:
Good point

Reply:
I will remove it. Thanks for the feedback @alexandrebodin